docs/fleet/cli.md

```diff
@@ -702,19 +702,103 @@ You can use the `module build start` or `module build local` commands to build y
 - Use `build start` to build or compile your module on a cloud build host that might offer additional platform support than you have access to locally.
 - Use `build local` to quickly test that your module builds or compiles as expected on your local hardware.
 
-To configure your module's build steps, add a `build` object to your [`meta.json` file](#the-metajson-file), including the following:
+To configure your module's build steps, add a `build` object to your [`meta.json` file](#the-metajson-file).
+Developers can either have a single build file for all platforms, or platform specific files:
+
+{{< tabs >}}
+{{% tab name="Single Build File" %}}
 
 ```json {class="line-numbers linkable-line-numbers"}
 "build": {
   "setup": "./setup.sh",                  // optional - command to install your build dependencies
```

Comment:
@michaellee1019 thinking including the `setup` line doesn't make sense here if build dependencies are installed with the build script but still want to show the option-- do you think that's ok? or would it be possible to have both?

Reply:
Yes both are supported at once. You could separate the file below to move the code up to `pip3 install -r requirements.txt` and then start the build script with `python3 -m PyInstaller...`. Up to you. It is better to have separate files for separation of concerns but likely easier for developers to copy/paste one file.

Reply:
Ok thanks so much @michaellee1019 -- going to show both methods since the expand hides unless they select so doesn't take up too much room